ai/src/main/java/org/triplea/ai/flowfield/map/FieldTerritory.java

```diff
@@ -0,0 +1,34 @@
+package org.triplea.ai.flowfield.map;
+
+import games.strategy.engine.data.Territory;
+import java.util.Collection;
+import java.util.HashMap;
+import java.util.Map;
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+import lombok.ToString;
+
+@Data
+@EqualsAndHashCode(exclude = "neighbors")
+public class FieldTerritory {
+  private Territory territory;
+  private long value;
+  @ToString.Exclude private final Map<Territory, FieldTerritory> neighbors = new HashMap<>();
```

Comment:
I'd rather this just be a `Set` since `FieldTerritory` has a reference to its `Territory`.  But when I did that, for some reason, I'd get duplicate neighbors in the set.